src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -496,6 +487,43 @@ public void cap(ClientReceiveCommandEvent event) {
         }
     }
 
+    @CommandFilter("CHGHOST")
```

Comment:
You've stuck this method between the CAP handling and the private capReq method used in it. Move it below the capReq method.